src/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    private final String name;
+    private boolean isComplete;
+
+    public Task(String name, boolean isComplete) {
+        this.name = name;
+        this.isComplete = isComplete;
+    }
+
+    public boolean getIsComplete() {
+        return this.isComplete;
```

Comment:
Don't use this.

Use this. Only when initialize member of the class in the constructor

Reply:
u do u resolve